proto/messages.proto

```diff
@@ -133,6 +135,21 @@ message UserDeleted {
   string id = 1;
 }
 
+message AgentSessionConnected {
+  Agent agent = 1;
+}
+
+message AgentSessionDisconnected {
+  Agent agent = 1;
+}
```

Comment:
What about just `AgentConnected` and `AgentDisconnected`?

"Session" or similar seems like a separate entity, so it would rather be `AgentSessionCreated`, but ideally we want to avoid session, because it already has multiple meanings :D

Reply:
`AgentConnected` already exists, so I can't use this name 😭

Reply:
and `AgentDisconnected` semantically, would be the opposite of current `AgentConnected`, which would work similarly like `UserDeleted`

Reply:
Ahh right, in retrospect the current one could be `AgentConnectionInit` or similar, but too late to change :p

So alternatively we can have `AgentUp`/`AgentDown`, `AgentJoined`/`AgentLeft`, or something verbose like `AgentConnectionCreated`/`AgentConnectionDeleted` (but deleted sounds weird). Happy to hear other ideas :)

Reply:
I like `AgentJoined` and `AgentLeft`